### PR TITLE
Adapt names to reviewed Kueue documentation

### PR DESCRIPTION
**Pull request by rbarberop**
The Kueue documentation has been updated recently.

This PR uses the same ResourceFlavor names than in documentation.

It also gives different names to jobs generated for both teams, so the output of the commands in the tutorial are more descriptive.

### CODE REVIEW

**Comment by rbarberop** [comment]
FYI @alculquicondor 
FYI @alizaidis

**Review by bourgeoisor — APPROVED** [review]
LGTM but before we merged we should have a CL ready to update https://cloud.google.com/kubernetes-engine/docs/tutorials/kueue-intro (for example I see some references of `cq` in there)

**Comment by alculquicondor** [comment]
(we can review, but do not merge yet)

**Comment by alculquicondor** [comment]
Oops, nvm. I thought this was about the new API fields. This can merge.

**Comment by rbarberop** [comment]
CL created @bourgeoisor

**Pull request merged**
